### ISSUE TITLE
iamterence.me is down

### ISSUE THREAD

**Issue by dvdsgl** [teren3ce/iamterence.me#1]
Is this online anywhere?

**Comment by teren3ce** [comment]
No, not yet. I have the designs done on Adobe XD but still making updates to it, also I decided to get iamterence.co instead

I could share with you if you don't mind, your input will be greatly appreciated
